packages-ts/gauntlet-terra-contracts/src/lib/utils.ts

```diff
@@ -27,3 +29,10 @@ export function deepCopy<T>(source: T): T {
       }, Object.create(Object.getPrototypeOf(source)))
     : (source as T)
 }
+
+export const fmtAddress = (address: AccAddress): string => {
+  const e = '\033['
+  return address == multisig
```

Comment:
Here, `multisig` is a global variable, so really hard to use this fn in any other context with any other multisig address than the one defined in the `@chainlink/gauntlet-terra-cw-plus` (env).

I'd suggest loading a "formatter" with different types of addresses it needs to recognize, and then it will format accordingly by comparing the input with the specified types and addresses in the registry. Potentially we also want to format token (LINK) addresses differently, etc.

Reply:
Yeah, I had in mind we'd want to add more at some point, just started with multisig since that's what they asked for.  I'll go ahead and add the others now... I think seeing clearly which contract is which will be very helpful in reducing careless user error.